front/src/NotificationWindow.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "styled-components";
+import faTimes from "@fortawesome/fontawesome-free-solid/faTimes";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+
+const Window = styled.div`
+  background-color : ${props => props.backgroundColor}
+  border-style: solid;
+  border-size: 4px;
+  width: 90%;
+  position: relative;
+  margin-left:auto;
+  margin-right:auto;
+`;
+
+const NotificationHeader = styled.p`
+  text-align: center;
+  font-size: 5vmin;
+`;
+
+const NotificationText = styled.p`
+  padding-left: 10px;
+  padding-right: 10px;
+  text-align: center;
+  font-size: 8vmin;
+`;
+
+const NotificationCloseButton = styled.button`
```

Comment:
You dont have to add `Notification` to every `sc` element.

Reply:
So should I rename them? It does not really matter imo.

Reply:
I would assume that all elements in `NotificationWindow` belong to the notification component. There is no need to prepend everything with `Notification`.